it/v2/endpoints/ListPaymentsAndAllocationDetailsControllerISpec.scala

```diff
@@ -163,9 +165,10 @@ class ListPaymentsAndAllocationDetailsControllerISpec extends IntegrationBaseSpe
         (BAD_REQUEST, "INVALID_DATE_FROM", BAD_REQUEST, FromDateFormatError),
         (BAD_REQUEST, "INVALID_DATE_TO", BAD_REQUEST, ToDateFormatError),
         (BAD_REQUEST, "INVALID_DATE_RANGE", BAD_REQUEST, RuleInvalidDateRangeError),
-        (FORBIDDEN, "REQUEST_NOT_PROCESSED", INTERNAL_SERVER_ERROR, InternalError),
+        (BAD_REQUEST, "INVALID_REQUEST", BAD_REQUEST, RuleInconsistentQueryParamsErrorListSA),
+        (BAD_REQUEST, "REQUEST_NOT_PROCESSED", BAD_REQUEST, BadRequestError),
```

Comment:
I think this should be FORBIDDEN instead of BAD_REQUEST ?

Reply:
The status code (403) doesn't reflect what we'd expect from the downstream system (409). The advice is that we should map 4XX to 4XX. So in this case 400.

Reply:
I don't follow - where is 409 from? Wouldn't we expect the downstream system to return a 403 (in line with our spec + DES spec)? 
<img width="758" alt="image" src="https://user-images.githubusercontent.com/106095575/213146978-f9aa659c-ba16-49fe-810c-0473d3a8a1eb.png">
Are we sure we shouldn't map 403 on the backend to 400 on our end ?